code/controllers/subsystem/mapping.dm

```diff
@@ -23,6 +24,8 @@ SUBSYSTEM_DEF(mapping)
 		fdel("data/next_map.txt") // Remove to avoid the same map existing forever
 	else
 		map_datum = new /datum/map/cyberiad // Assume cyberiad if non-existent
+		*/
+	map_datum = new /datum/map/test_desert
```

Comment:
You can put this on L25 and have it still work, and just disable the verb side if you want your code to be more modular.